components/ai_chat/resources/untrusted_conversation_frame/untrusted_conversation_frame.tsx

```diff
@@ -10,12 +10,70 @@ import '$web-components/app.global.scss'
 import '$web-common/defaultTrustedTypesPolicy'
 import ConversationEntries from './components/conversation_entries'
 import { UntrustedConversationContextProvider } from './untrusted_conversation_context'
+import UntrustedConversationFrameAPI from './untrusted_conversation_frame_api'
 
 import '../common/strings'
 
 setIconBasePath('chrome-untrusted://resources/brave-icons')
 
 function App() {
+  // Iframe drag detection with proper cleanup
+  React.useEffect(() => {
+    let dragCounter = 0
+    const api = UntrustedConversationFrameAPI.getInstance()
```

Comment:
I think let's reexpose the functions we need from the `UntrustedConversationFrameContext` and access if from there. We try and access all the mojom APIs through it to make things easier to test / use from storybook